pkg/cmd/server/server.go

```diff
@@ -458,6 +458,9 @@ var defaultRestorePriorities = []string{
 	"customresourcedefinitions",
 	"namespaces",
 	"storageclasses",
+	"volumesnapshotclass.snapshot.storage.k8s.io",
```

Comment:
please update the field comment to explain why these resources are where they are in the list